ReportingServicesTools/Functions/CatalogItems/Write-RsCatalogItem.ps1

```diff
@@ -122,11 +122,11 @@ function Write-RsCatalogItem
                 ) -or
                 (
                     $itemType -eq "Resource" -and
-                    $item.Extension -notin ('.png', '.jpg', '.jpeg')
+                    $item.Extension -notin ('.png', '.jpg', '.jpeg','.gif','.bmp')
                 )
             )
             {
-                throw "Invalid item specified! You can only upload Report, DataSource, DataSet and jpg/png files using this command!"
+                throw "Invalid item specified! You can only upload Report, DataSource, DataSet and jpg/png/gif/bmp files using this command!"
```

Comment:
please also add a test, check [https://github.com/Microsoft/ReportingServicesTools/blob/c92b8b0a06819da698dc10bc21258d79d43aafc1/Tests/CatalogItems/Write-RsCatalogItem.Tests.ps1#L185](https://github.com/Microsoft/ReportingServicesTools/blob/c92b8b0a06819da698dc10bc21258d79d43aafc1/Tests/CatalogItems/Write-RsCatalogItem.Tests.ps1#L185) for reference